### PR TITLE
README: improved formatting

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
 # Homebrew webpage
 
-The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for the index is at layouts/index.html.
+The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for
+the index is at `_layouts/index.html`.
 
 ## Translations
+
```
Comment (mikemcquaid) [inline README.md]: I prefer to not add newlines after headings.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,20 @@
 # Homebrew webpage
 
-The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for the index is at layouts/index.html.
+The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for
+the index is at `_layouts/index.html`.
```
Comment (mikemcquaid) [inline README.md]: Generally I don't see the advantage in hard-breaking lines in Markdown but don't mind doing it here.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
 
 ## Translations
+
 If you want to add a new translation, follow these steps:
 
-1. In _config.yml append this:
+1. In `_config.yml` append this:
 
-	```
+	```yml
```
Comment (mikemcquaid) [inline README.md]: Good find :

File: README.md (hunk 1 above)
Comment (bfontaine) [inline README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (bfontaine) [inline README.md]: I’m sorry I don’t understand what you want here. Do you want me put each sentence on its own line? I broke the line to keep the text under 80 chars but this doesn’t change the way how the text is displayed.

Comment (mikemcquaid) [inline README.md]: You can leave it as-is. I guess my point is I don't think keeping it under 80 chars is necessary for Markdown but I'll merge this as-is.

Comment (bfontaine) [inline README.md]: Ok, thanks for the clarification.

**Pull request merged**